examples/containerlab/.topology.clab.yml.bak

```diff
@@ -0,0 +1,22 @@
+name: bonding
```

Comment:
I just noticed this file shouldn't be here, but the comment above then apply to the real file